responsibleai/responsibleai/managers/explainer_manager.py

```diff
@@ -48,7 +49,9 @@ class ExplainerManager(BaseManager):
     """Defines the ExplainerManager for explaining a model."""
 
     def __init__(self, model, initialization_examples, evaluation_examples,
```

Comment:
Can you add type hints for model, initialization_examples, evaluation_examples and target_column?